cmd/verrazzano-operator/main.go

```diff
@@ -128,35 +130,41 @@ func main() {
 		glog.Fatalf("Error creating the controller: %s", err.Error())
 	}
 
-	//
+	apiServerExited := make(chan bool)
+
 	// start the REST API Server (for web ui)
-	//
+	if startAPIServer {
+		startRestAPIServer(controller.ListerSet(), apiServerExited)
+	}
 
+	if startController {
+		// start the controller
```

Comment:
What if `startAPIServer==false && startController==true`?  If the `apiServerExited` channel never gets `<- true`, will it ever get past `<-apiServerExited`?

Reply:
Good catch Tom - I actually don't need the startAPIServer flag - my current need is to only be able to tell the operator to start or not start the controller. I added this flag for symmetry.

Rather than introduce more code to handle the case you describe, for a case I don't even need, I've now removed the startAPIServer flag altogether. Hopefully the build passes :-)